src/Geosuggest.tsx

```diff
@@ -594,6 +602,9 @@ export default class extends React.Component<IProps, IState> {
         onPrev={this.onPrev}
         onSelect={this.onSelect}
         onEscape={this.hideSuggests}
+        isSuggestHidden={this.state.isSuggestsHidden}
```

Comment:
The state key is called `isSuggestsHidden`, and you pass down `isSuggestHidden`. Maybe we keep the same for consistency?

Reply:
sure, it's a typo